src/ServiceControl.Audit/ServiceControl.Audit.csproj

```diff
@@ -35,6 +39,12 @@
     <PackageReference Include="Particular.CodeRules" Version="0.2.1" PrivateAssets="All" />
   </ItemGroup>
 
+  <ItemGroup>
```

Comment:
Not sure I see the benefit of having this nested like this, since it just hides one of the two files. Removing the `ItemGroup` would also simplify the changes in the PR.